packages/sdk/src/sdk/api/users/UsersApi.ts

```diff
@@ -431,34 +479,4 @@ export class UsersApi extends GeneratedUsersApi {
       ...advancedOptions
     })
   }
-
-  /** @hidden
-   * Update an encrypted email for a user
-   */
-  async updateEmail(params: EmailRequest, advancedOptions?: AdvancedOptions) {
```

Comment:
this isn't being used right now, will merge updating logic with the `shareEmail` function in the future